libraries/chain/include/eosio/chain/block_header_state.hpp

```diff
@@ -150,13 +149,10 @@ using block_header_state_ptr = std::shared_ptr<block_header_state>;
 
 }
 
-FC_REFLECT_ENUM( eosio::chain::finalizer_policy_tracker::state_t, (proposed)(pending))
-
-FC_REFLECT( eosio::chain::finalizer_policy_tracker, (state)(policy))
-
 FC_REFLECT( eosio::chain::block_header_state, (block_id)(header)
             (activated_protocol_features)(core)(active_finalizer_policy)
-            (active_proposer_policy)(proposer_policies)(finalizer_policies)
-            (finalizer_policy_generation)(last_pending_finalizer_policy_digest)(header_exts))
+            (active_proposer_policy)(proposer_policies)(proposed_finalizer_policies)
+            (pending_finalizer_policy)(finalizer_policy_generation)
+            (last_pending_finalizer_policy_digest)(header_exts))
```

Comment:
Not part of your chnages, but I don't think `header_exts` should be included here. They are recreated in the `fork_database` when the `block_state` is read in.

Reply:
https://github.com/AntelopeIO/spring/pull/307/commits/13452205fa72e0c35be82fa6f430c790ebc3dc62